lib/asyn-thread.c

```diff
@@ -69,6 +69,12 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
+#ifdef _WIN32
+#include "system_win32.h"
+/* set in win32_init() */
+extern bool Curl_isWindows8OrGreater;
```

Comment:
This should be an extern placed in `system_win32.h` to match the variable which is declared in `system_win32.c`.

You don't need to protect this include with an `#ifdef` as the header itself deals with that.

Finally: you placed this include *below* the includes just above which is said to be used as the last include files in this file: move this include to above those.

Reply:
updated